Assets/script.js

```diff
@@ -152,176 +145,208 @@ var getEventsSearch = async function (city)
     var eventSearchParams = `?apikey=${eventsAPIKey}&size=60&sort=date,name,asc`;
     eventSearchParams = !cityMod == "" ? eventSearchParams + `&city=${cityMod}` : eventSearchParams;
     //append the keyword(s) to the url if they are present
-    var evtKeyword = document.getElementById("eventsKeywordInput").value.trim();
+    var evtKeyword = document.getElementById("destinationInput").value.trim();
```

Comment:
This is the keyword search for events. It is not the destination. The element id is eventsKeywordInput.

Reply:
I consolidated our search button's, since we no longer have the origin input we only need one. Which is why I changed the dvtKeyword to the destinationInput. I